app/workers/education_form/process10203_submissions.rb

```diff
@@ -0,0 +1,185 @@
+# frozen_string_literal: true
+
+require 'evss/gi_bill_status/service'
+require 'sentry_logging'
+
+module EducationForm
+  DENIED = 'denied'
+  PROCESSED = 'processed'
+  INIT = 'init'
+
+  class FormattingError < StandardError
+  end
+
+  class Process10203SubmissionsLogging < StandardError
+  end
+
+  class Process10203Submissions
+    include Sidekiq::Worker
+    include SentryLogging
+    sidekiq_options queue: 'default',
+                    unique_for: 30.minutes,
+                    retry: 5,
+                    backtrace: true
+
+    # Get all 10203 submissions that have a row in education_stem_automated_decisions
+    def perform(
+      records: EducationBenefitsClaim.joins(:education_stem_automated_decision).includes(:saved_claim).where(
+        saved_claims: {
+          form_id: '22-10203'
+        }
+      )
+    )
+      return unless Flipper.enabled?(:stem_automated_decision) && evss_is_healthy?
```

Comment:
Does returning here add it back to the retry queue? Or do we not want to retry if evss isn't healthy?

Reply:
made it return false like other sidekiq jobs when they shouldn't run

Reply:
I assume you mean falsey. But I was just wanting to make sure it was okay to skip this if evss is down. In most cases, I'd think you'd want it to re-run when EVSS came back up.

As long as you're aware. :+1:

Reply:
The job needs EVSS up so that an user's entitlement data can be pulled and checked